src/main/java/design/MazeDesignerController.java

```diff
@@ -0,0 +1,52 @@
+package design;
+
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+
+public class MazeDesignerController {
+    MazeDesignerInteractor md = new MazeDesignerInteractor();
+    MazeDesignerPresenter mdp = new MazeDesignerPresenter();
```

Comment:
I don't think your controller should have access to the presenter since it should be taking the input, sending it to the interactor and have the interactor respond and send the necessary information back up to the presenter through the out boundary